### PR TITLE
docs: correct punctuations in README.md

### PR DESCRIPTION
**Pull request by yashrajbharti**
Corrected all grammatical errors and punctuations.

**Summary**

**Related Issues/PRs**

### CODE REVIEW

**Comment by yashrajbharti** [comment]
I signed it!

File: readme.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ lighthouse -GA=./gmailartifacts https://gmail.com
 
 #### Notes on Error Reporting
 
-The first time you run the CLI you will be prompted with a message asking you if Lighthouse can anonymously report runtime exceptions. The Lighthouse team uses this information to detect new bugs and avoid regressions. Opting out will not affect your ability to use Lighthouse in any way. [Learn more](https://github.com/GoogleChrome/lighthouse/blob/master/docs/error-reporting.md).
+The first time you run the CLI, you will be prompted with a message asking you if Lighthouse can anonymously report runtime exceptions. The Lighthouse team uses this information to detect new bugs and avoid regressions. Opting out will not affect your ability to use Lighthouse in any way. [Learn more](https://github.com/GoogleChrome/lighthouse/blob/master/docs/error-reporting.md).
```
Comment (exterkamp) [inline readme.md]: I disagree with this.  I think it is fine without the comma, the beginning isn't really a distinct introductory phrase that would warrant a comma.  I don't think :man_shrugging:

If the first part was reworded to be more distinct I could see it.

File: readme.md [hunk 2]
```diff
@@ -324,7 +324,7 @@ Yes! Details in [Lighthouse configuration](./docs/configuration.md).
 ### How does Lighthouse use network throttling, and how can I make it better?
 
 Good question. Network and CPU throttling are applied by default in a Lighthouse run. The network
-attempts to emulate slow 4G and the CPU is slowed down 4x from your machine's default speed. If you
+attempts to emulate slow 4G, and the CPU is slowed down 4x from your machine's default speed. If you
```
Comment (exterkamp) [inline readme.md]: I think I'm getting caught up on the `slow 4G` wording as well.  I'd add a little more context to it.
```
The network attempts to emulate a slow 4G network, and the CPU is slowed down 4x from 
the machine's default speed.
```

**Comment by paulirish** [comment]
addressed feedback.

in a separate PR i'll rewrite that throttling paragraph since it's incorrect now.

**Pull request merged**